config/secrets.yml

```diff
@@ -31,12 +29,9 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  forest_env_secret: <%= ENV["FOREST_ENV_SECRET"] %>
-  forest_auth_secret: <%= ENV["FOREST_AUTH_SECRET"] %>
-  git_hub_client_id: <%= ENV["GIT_HUB_CLIENT_ID"] %>
-  git_hub_client_secret: <%= ENV["GIT_HUB_CLIENT_SECRET"] %>
```

Comment:
These two GitHub production env vars actually remain.

Reply:
These Forest admin ones need to be removed:

```
   forest_env_secret: <%= ENV["FOREST_ENV_SECRET"] %>
   forest_auth_secret: <%= ENV["FOREST_AUTH_SECRET"] %>
```

The GitHub ones are fine, now.